src/jit/importer.cpp

```diff
@@ -7492,17 +7492,6 @@ GenTreePtr                Compiler::impFixupCallLongReturn(GenTreePtr     call,
     unsigned retRegCount = retTypeDesc->GetReturnRegCount();
     // must be a long returned in two registers
     assert(retRegCount == 2);
-
```

Comment:
updated the function name and header. Should we also change the logic of impFixupCallStructReturn to call this function (and make it available to more than just TARGET_X86)?